kickstarter/android-oss
# Reward disabled state

# What ❓
- Added logic to configure the disabled state of the pledge button in the `HorizontalRewardViewHolder`

# How to QA? 🤔
- Go to a project with rewards and when is a reward isn't available you will see the pledge button with the following text `No longer available` with a transparent green shade.

# Story 📖
[Reward disabled state](https://trello.com/c/0NMN6WCr/1393-reward-disabled-state)

# See 👀
<img src=https://user-images.githubusercontent.com/16387538/58964628-39afe000-877d-11e9-82e6-176c6a43acbc.jpg width=220/>

![disabled](https://user-images.githubusercontent.com/16387538/58964700-52b89100-877d-11e9-82bf-635673b2b832.gif)
